components/raftstore/src/store/fsm/store.rs

```diff
@@ -787,25 +788,49 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport> PollHandler<PeerFsm<EK, ER>, St
     }
 
     fn end(&mut self, peers: &mut [Box<PeerFsm<EK, ER>>]) {
-        for peer in peers {
+        for peer in peers.iter_mut() {
```

Comment:
Why `iter_mut`?

Reply:
Because `peers` will be moved if using `into_iter` implicitly.